requirements/prd.txt

```diff
@@ -1,4 +1,5 @@
 ray[default]==2.8.0
+async-timeout==4.0.3
```

Comment:
this seems to be a missing transitive dependency of the ray dashboard component that somehow is missing in py311 case. Ideally we can remove this explicit spec later

Reply:
Shouldn't this also be added to the ``pyproject.toml``?